components/manager-ip-setter/scripts/manager-ip-setter.sh

```diff
@@ -19,17 +19,27 @@ function set_manager_ip() {
   /usr/bin/sed -i -e "s/REST_HOST=.*/REST_HOST="'"'"${ip}"'"'"/" /etc/sysconfig/cloudify-mgmtworker
   /usr/bin/sed -i -e "s#MANAGER_FILE_SERVER_URL="'"'"https://.*:53333/resources"'"'"#MANAGER_FILE_SERVER_URL="'"'"https://${ip}:53333/resources"'"'"#" /etc/sysconfig/cloudify-mgmtworker
 
+  echo "Updating cloudify-manager (rest-service).."
+  /usr/bin/sed -i -e "s#amqp_address: '.*:5671/'#amqp_address: '${ip}:5671/'#" /opt/manager/cloudify-rest.conf
+  /usr/bin/sed -i -e "s#file_server_url: 'https://[^:]*:\(.*\)#file_server_url: 'https://${ip}:\1#" /opt/manager/cloudify-rest.conf
+
   echo "Updating broker_config.json.."
   /usr/bin/sed -i -e "s/"'"'"broker_hostname"'"'": "'"'".*"'"'"/"'"'"broker_hostname"'"'": "'"'"${ip}"'"'"/" /opt/mgmtworker/work/broker_config.json
 
   echo "Updating broker_ip in provider context.."
-  /opt/cloudify/manager-ip-setter/update-provider-context.py ${ip}
+  /opt/mgmtworker/env/bin/python /opt/cloudify/manager-ip-setter/update-provider-context.py ${ip}
 
   echo "Creating internal SSL certificates.."
-  /opt/cfy/embedded/bin/python /opt/cloudify/manager-ip-setter/create-internal-ssl-certs.py ${ip}
+  /opt/mgmtworker/env/bin/python /opt/cloudify/manager-ip-setter/create-ssl-certs.py ${ip}
   
-  echo "Restarting nginx.."
+  echo "Restarting services.."
+  # Restarting all (except postgres) to avoid issues with not correctly reloading SSL certs
   systemctl restart nginx
+  systemctl restart cloudify-rabbitmq
```

Comment:
Whoops, I thought I removed the restarts from elsewhere.

I ran into issues on one of the build attempts where I had to restart some services that weren't restarted because at least one appeared to be caching a cert, so I put in restarts on all of the services we use so that even if we add extra logic to the IP setting at a later date, things should just work (tm), rather than having to decide specifically which services need restarting each time.

I'll remove the earlier RabbitMQ and retest.